README.md

```diff
@@ -54,17 +55,54 @@ and certificates needed to talk to the peer and orderer node.
 docker exec -it cli bash
 ```
 
-2. Next install the chaincode. The chaincode has already been mounted into the
+2. package the chaincode. The chaincode has already been mounted into the
 peer container. You can see more details in [`docker-compose.yml`](./docker-compose.yml).
 ```
-peer chaincode install -n fabcar -v 1.0 -l node -p /opt/gopath/src/github.com/chaincode
+peer lifecycle chaincode package fabcar.tar.gz --path /opt/gopath/src/github.com/chaincode/ --lang node --label fabcar1
 ```
 
-3. Instantiate the chaincode. At the end of this
-you should see a new container appear which is the chaincode container.
+1. install the chaincode using the package.
+```
+peer lifecycle chaincode install fabcar.tar.gz
+```
+You should see a nodeenv container run to completion.
+
+1. query the peer for the installed chain code to get the Package ID:
+```
+$ peer lifecycle chaincode queryinstalled
+Installed chaincodes on peer:
+Package ID: fabcar1:cb90821c789b3584ed7c388269a85e51997ac3042908ba380aabbc265f52af05, Label: fabcar1
+```
+
+1. approve the chaincode for running.
+```
+peer lifecycle chaincode approveformyorg --channelID mychannel --name fabcar --version 1 --sequence 1 --package-id fabcar1:433b167e5a9d9b3dd089ca4e4a9f757f0f2effbc1255e9a7b3a2313860a797ff
```

Comment:
need the package ID, but it'll change every time, not sure if we should variable it out here or leave this in as it is.